egs/wsj/s5/steps/nnet3/lstm/train.sh

```
@@ -243,6 +252,8 @@ else
   ivector_dim=$(feat-to-dim scp:$online_ivector_dir/ivector_online.scp -) || exit 1;
 fi
 
+[ $ivector_interval -gt $chunk_width ] && ivector_interval=0
```

better to throw an error and die.